src/object/player.cpp

```diff
@@ -774,7 +774,7 @@ Player::handle_horizontal_input()
     }
     else {
       // give Tux tighter air control
-      ax *= 2.f;
+      ax *= 1.5f;
```

Comment:
This seems to be unrelated to climbing.

Reply:
> This seems to be unrelated to climbing.

Oops, I was messing with Tux's velocity and forgot to reset it.